.circleci/test-github-repo.sh

```diff
@@ -23,3 +23,10 @@ terminus build:project:create -n "$SOURCE_COMPOSER_PROJECT" "$TERMINUS_SITE" --t
 terminus site:info "$TERMINUS_SITE"
 # Confirm that the Github project was created
 git clone "$CLONE_URL" "$TARGET_REPO_WORKING_COPY"
+
+# Replace with right docker image.
+cd $TARGET_REPO_WORKING_COPY
```

Comment:
Temporarily replace the docker image in the test project to use a tag that contains php 7.4 and t3.